presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -194,6 +215,9 @@ public TaskExecutor(
                 guaranteedNumberOfDriversPerTask,
                 maximumNumberOfDriversPerTask,
                 taskPriorityTracking,
+                DEFAULT_INTERRUPT_RUNAWAY_SPLITS_TIMEOUT,
```

Comment:
why not `config.getInterruptRunawaySplitsTimeout()` directly?

Reply:
This is a test constructor and there is no accessible `TaskManagerConfig` I think? Or you mean instantiate the config and then request the value? I can do that if it is what you are thinking.

Reply:
Yes, given this is for test only, we can do that